Products/Zuul/catalog/model_catalog.py

```diff
@@ -125,6 +125,13 @@ def getRID(self):
         """Return the record ID for this object."""
         return getattr(self, "uuid")
 
+    def to_dict(self, idxs=[]):
```

Comment:
It is not a good practice to pass lists as default parameters to methods bc it has side effects. http://docs.python-guide.org/en/latest/writing/gotchas/